infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/main.tf

```
@@ -131,6 +132,7 @@ module "prow_build_cluster" {
   bigquery_location = local.bigquery_location
   is_prod_cluster   = "true"
   release_channel   = "STABLE"
+  enable_node_local_dns_cache = var.enable_node_local_dns_cache
```

I may be wrong but I think it's : 
```suggestion
  enable_node_local_dns_cache = lcoal.enable_node_local_dns_cache
```

---

my bad, bad copy/paste :)

---

Fixed